File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/AmputeeAssessmentFormViewModel.ts
//$0D050CFF
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { AmputeeAssessmentForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class AmputeeAssessmentFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.AmputeeAssessmentFormViewModel, Core';
    @Type(() => AmputeeAssessmentForm)
    public _AmputeeAssessmentForm: AmputeeAssessmentForm = new AmputeeAssessmentForm();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/AmputeeAssessmentFormViewModel.cs
//$BFC4833D
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class AmputeeAssessmentFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class AmputeeAssessmentFormViewModel : BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var amputeeAssessmentFormService = new AmputeeAssessmentFormServiceController();
            amputeeAssessmentFormService.AmputeeAssessmentFormInternal(this, objectContext);
            return (AmputeeAssessmentForm)objectContext.GetObject(this._AmputeeAssessmentForm.ObjectID, "AmputeeAssessmentForm");
        }
    }
}
